pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -194,8 +201,7 @@
       "playbook_file": "./ansible/provision.yaml",
       "user": "{{user `ssh_username`}}",
       "ansible_env_vars": [
-        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes -o HostkeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa'",
-        "ANSIBLE_REMOTE_TEMP='/tmp/.ansible/'"
+        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes -o HostkeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa'"
```

Comment:
Similar to https://github.com/mesosphere/konvoy-image-builder/pull/559/files#r1010945066, to preserve existing behavior we would want something like:
```suggestion
        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes -o HostkeyAlgorithms=+ssh-rsa -o PubkeyAcceptedAlgorithms=+ssh-rsa'"
        "ANSIBLE_REMOTE_TEMP='{{ remote_folder }}/.ansible/'"
```

Reply:
fixed.